lib/msal-core/src/utils/Constants.ts

```diff
@@ -107,6 +107,8 @@ export const AADTrustedHostList =  {
     "login.microsoftonline.us": "login.microsoftonline.us"
 };
 
+export const B2CTrustedHostList = {};
```

Comment:
I would consider moving this to AuthorityFactory or B2CAuthority since it isn't actually a constant.

Reply:
I'm also not a huge fan of having a global variable like this, I think it should be a property or field of an object or class

Reply:
Moved this to AuthorityFactory but not sure how to avoid making it global since it needs to be read from the static detectAuthorityFromUrl function.